src/components/SignerPanel/ActionPathsContent.js

```diff
@@ -24,13 +30,56 @@ class ActionPathsContent extends React.Component {
       pretransaction
     )
   }
-  renderDescription(showPaths, { description, name, to }) {
+  renderDescription(
+    showPaths,
+    { description, name, to, annotatedDescription }
+  ) {
     return (
       <React.Fragment>
-        <p>This transaction will {showPaths && 'eventually'} perform:</p>
-        <p style={{ margin: '10px 0' }}>
-          {description ? `"${description}"` : 'an action'}
-        </p>
+        <p>This transaction will {showPaths && 'eventually'} perform</p>
+        <div style={{ margin: '10px 0 10px 15px' }}>
+          {annotatedDescription
+            ? annotatedDescription.map(({ type, value }, index) => {
+                if (type === 'address') {
+                  return (
+                    <IdentityBadge
```

Comment:
:/ For some reason this doesn't like to render in Chrome (might just be my current version). The entire `Info.Action` component is acting like it has `display: none`:

<img width="1167" alt="screen shot 2018-12-12 at 8 12 30 pm" src="https://user-images.githubusercontent.com/4166642/49892869-557b9e80-fe4a-11e8-80f5-35ea58e97a02.png">

<img width="1440" alt="screen shot 2018-12-12 at 8 12 56 pm" src="https://user-images.githubusercontent.com/4166642/49892872-590f2580-fe4a-11e8-8c11-ccc6c1989057.png">


cc @bpierre could you check this? The DOM nodes are still there, but nothing shows up until I toggle a css property.